docroot/modules/custom/hs_paragraph_types/modules/hs_slide/config/install/core.entity_form_display.paragraph.hs_slide.default.yml

```diff
@@ -0,0 +1,62 @@
+langcode: en
```

Comment:
Is there a reason why the hs_slide and hs_slideshow modules are not enabled by default?

Reply:
I'm not sure what all they want enabled by default just yet. its likely it will go that way

Reply:
Just checking. This is fine with me.